### PR TITLE
fix broken intradoc links

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -2883,7 +2883,7 @@ impl Build {
     /// Get the archiver that's in use for this configuration.
     ///
     /// This will return a result instead of panicing;
-    /// see [`get_archiver()`] for the complete description.
+    /// see [Self::get_archiver] for the complete description.
```
Comment (NobodyXu) [inline src/lib.rs:2886]: I thought the infra link needs to be quoted by `

Comment (Be-ing) [reply src/lib.rs:2886]: That is not necessary, it's just for formatting.

Comment (NobodyXu) [reply src/lib.rs:2886]: thx, though I prefer it to be quoted

**Review by NobodyXu — APPROVED** [review]
Thank you!

**Pull request merged**